src/Lykke.Job.CandlesProducer/Settings/CandlesGeneratorSettings.cs

```diff
@@ -5,5 +5,6 @@ namespace Lykke.Job.CandlesProducer.Settings
     public class CandlesGeneratorSettings
     {
         public TimeSpan MinCacheAge { get; set; }
```

Comment:
Could we remove this obsolete settings property?

Reply:
Fixed by property removal.